po.js

```diff
@@ -175,6 +175,7 @@ PO.prototype.replaceAll = function (string, find, replace) {
 }
 
 PO.prototype.fixString = function (savedString) {
+    savedString = this.replaceAll(savedString, "\\", '\\\\');
```

Comment:
There is currently one string that legitimately contains a backslash (\). If the above line is not present the translated string will just never make it into the translation and thus will be presented as untranslated whenever you generate the PO file again.